src/components/workspace/Workspace.js

```diff
@@ -712,6 +862,7 @@ export default class Workspace extends Base {
 
   renderComponents(flows) {
     this.workspace.innerHTML = '';
+    this.ac(this.workspace, this.svg);
```

Comment:
`this.ac(this.workspace, [ this.svg, flows.map((component) => this.placeComponent(component))]);`